.github/workflows/ci.yml

```diff
@@ -214,7 +214,9 @@ jobs:
           path: node_modules
           key: ${{ runner.os }}-cache-node-modules-${{ hashFiles('**/package-lock.json') }}
       - name: Build assets
-        run: NODE_ENV=production npm run build:iframes
```

Comment:
Pour info : on utilisait l'environment 1production` mais comme le fichier n'était pas présent on était sur la config par default : et avant cette PR la baseUrl par default était `https://mes-aides.1jeune1solution.beta.gouv.fr`

Reply:
J'ai vérifié et non car on demande pas l'étape `build` pour l'`iframe_build_control` du coup on avait le `No specific configuration for production` ([exemple](https://github.com/betagouv/aides-jeunes/actions/runs/6170197068/job/16746026384))